packages/config-service/src/services/globalConfig.ts

```diff
@@ -742,5 +763,10 @@ export class GlobalConfig {
       required: false,
       defaultValue: null,
     },
-  };
+} satisfies { [key: string]: ConfigProperty };
```

Comment:
for this approach to work, `_CONFIG` must be declared `as const`

```suggestion
} as const satisfies { [key: string]: ConfigProperty };
```

Reply:
@acuarica Should I go ahead to add the suggested changes as specified? and is there things else that need to be added.

Reply:
make sure it compiles correctly locally, see failing job here https://github.com/hashgraph/hedera-json-rpc-relay/pull/3350#issuecomment-2608208535 to find out the command to run.

Reply:
> Should I go ahead to add the suggested changes as specified?

Yes, that would be great, thanks.